Sources/Kitura/RouterRequest.swift

```diff
@@ -32,7 +32,7 @@ public class RouterRequest: SocketReader {
     /// The hostname of the request
     ///
     public var hostname: String {
-        if  let host = headers["host"] {
+        if  let host = headers.get("host")?.first {
```

Comment:
Wouldn't the use of subscript (```header["host"]```) be better here? Since header is storing and returning values as string array, setting it with ```header["host"] = [value]``` for a single value shouldn't be a problem.